cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -82,36 +111,38 @@ metadata:
   name: token-${safe_username}
 type: Opaque
 EOF
-  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" &
+  create-resource-from-string "${secretyaml}" 100 10 "Secret-for-token-for-user-${username}" "${SYSTEM_NAMESPACE}" &
```

Comment:
Addon-updater does not deal with secrets. You have to take care of secrets in the default namespace here.